lib/utils.js

```diff
@@ -15,6 +15,10 @@ export const normalize = (champion, imageCdnUrl) => {
   return champion;
 };
 
+export const normalizeChampion = (champion, imageCdnUrl) => {
+  return normalize(champion, imageCdnUrl);
```

Comment:
Since `normalize` is used only within this file, it no longer has to be exported within the public API of this file.